cmake/Platform/Windows/Packaging/Tests/test_installer.py

```diff
@@ -0,0 +1,248 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+Pytest test configuration file.
+
+"""
+import pytest
+from pathlib import Path
+from subprocess import run, TimeoutExpired
+from time import sleep
+import json
+import os
+import shutil
+
+@pytest.fixture(scope="session")
+def installer_path(request):
+    return Path(request.config.getoption("--installer-path")).resolve()
+
+@pytest.fixture(scope="session")
+def install_root(request):
+    return Path(request.config.getoption("--install-root")).resolve()
+
+@pytest.fixture(scope="session")
+def project_path(request):
+    # MSBuild doesn't want us to output in the temp directory MSB8029
+    # so we use a pre-defined folder outside of %TMP%
+    #return tmpdir_factory.mktemp("TestProject")
+    path = Path(request.config.getoption("--project-path")).resolve()
+    yield path
+
+    # wait a few seconds for processes to quit
+    sleep(5)
+
+    shutil.rmtree(path)
+
+@pytest.fixture(scope="session")
+def cmake_path(install_root):
+    return install_root / 'cmake' / 'runtime' / 'cmake-3.20.2-windows-x86_64' / 'bin' / 'cmake.exe'
```

Comment:
will likely change this to use whatever cmake runtime is included with the installer

Reply:
Maybe a glob pattern search for `cmake/runtime/cmake*/bin/cmake.exe` will help make this simpler? Agree that we shouldn't have rigid paths for this.